### ISSUE TITLE
lodash is imported globaly when importing ANYTHING from 'native-base'

### ISSUE THREAD

**Issue by marxus85** [GeekyAnts/NativeBase#1000]
### packages versions
"native-base": "^2.1.5",
"react": "16.0.0-alpha.12",
"react-native": "0.45.1",

### expected behaviour
import { List } from 'native-base'
const something = _.noop
-> error: Can't find variable: _

### actual behaviour
import { Icon } from 'native-base'
const something = _.noop 
-> it just works... even without including lodash in the code... native-base somehow include it globaly...that should not be the case

**Comment by horiuchie** [comment]
I have reproduced as well.

Also I have found it works as expected if I import from `NativeBase` directory cloned referring to [Contributing to NativeBase](https://github.com/GeekyAnts/NativeBase/blob/master/CONTRIBUTING.md#contributing-to-nativebase).

I guess the code exporting lodash to global is included in `dist` directory that is created by `npm run transpile`.

So this might be due to babel configuration when transpiling ( like  lodash/lodash#1798 ).
